BankManagerConsoleApp/Model/Console/BankProcess.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  BankProcess.swift
+//  BankManagerConsoleApp
+//
+//  Created by 우롱차, 민성 on 2022/04/28.
+//
+
+import Foundation
+
+struct BankProcess {
+    private enum BankProcessConstant {
+        static let minClient = 10
+        static let maxClient = 30
```

Comment:
변수 상수 네이밍은 어떤 값을 표현하는지 명확하게 표현해주시는걸 권장드립니다. 예를 들어 clerkCount와 같이 minClient에 최소 고객의 수를 의미하도록 Count를 붙여줘서 숫자 데이터임을 명확하게 해주는걸 추천드립니다.
```suggestion
        static let minClientCount = 10
        static let maxClientCount = 30
```

Reply:
항상 네이밍은 어렵네요... 감사합니다.